oemof_b3/tools/data_processing.py

```diff
@@ -142,6 +142,26 @@ def load_b3_scalars(path, sep=";"):
     return df
 
 
+def load_unstacked_b3_scalars(path, sep=";"):
```

Comment:
Have you tested this function? So far it is not used anywhere right?

Reply:
You are right. It is not used and not tested. I guess I needed it when playing around with the functions, and it may be used sometimes. But currently not. Better delete it?